plugins/repository-hdfs/build.gradle

```diff
@@ -116,14 +115,7 @@ tasks.named("dependencyLicenses").configure {
 
 thirdPartyAudit {
   ignoreViolations(
-    // uses internal java api: sun.misc.Unsafe
-    'com.google.protobuf.MessageSchema',
-    'com.google.protobuf.UnsafeUtil',
-    'com.google.protobuf.UnsafeUtil$1',
-    'com.google.protobuf.UnsafeUtil$Android32MemoryAccessor',
-    'com.google.protobuf.UnsafeUtil$Android64MemoryAccessor',
-    'com.google.protobuf.UnsafeUtil$JvmMemoryAccessor',
-    'com.google.protobuf.UnsafeUtil$MemoryAccessor'
+
   )
 }
```

Comment:
Do we need this anymore?

Reply:
Yup we do. We are removing these violations from the plugin.

Reply:
Sorry if I wasn't clear. I meant do we need `thirdPartyAudit` task here anymore in this build.gradle? Can we remove the task from this build.gradle

Reply:
Ah I missed that. It makes sense.